components/asana/sources/new-tag/new-tag.mjs

```diff
@@ -0,0 +1,50 @@
+import asana from "../../asana.app.mjs";
+
+export default {
+  key: "asana-new-tag",
+  type: "source",
+  name: "New Tag",
+  description: "Emit new event for each tag created in a workspace.",
+  version: "0.0.2",
+  dedupe: "unique",
+  props: {
+    asana,
+    workspace: {
+      label: "Workspace",
+      description: "Gid of a workspace.",
+      type: "string",
+      propDefinition: [
+        asana,
+        "workspaces",
+      ],
+      optional: true,
+    },
+    db: "$.service.db",
+    http: {
+      type: "$.interface.http",
+      customResponse: true,
+    },
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
```

Comment:
I think you can also create a webhook making use of [tag resource type](https://developers.asana.com/docs/tag) instead of polling the tags

Reply:
Yes, I tried, but don't worked, this is the reason for listing all tags to emit the events.

I thing this is outdated in the API docs.